overcast_to_sqlite/html/page.py

```
@@ -26,6 +27,57 @@
     return "".join(result)
 
 
+class HTMLTagFixer(HTMLParser):
+    def __init__(self):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)